### PR TITLE
Costmetic fixes.

### PR DESCRIPTION
**Pull request by montera82**
Some Costmetic change for the snap 
Fixes https://github.com/celo-org/celo-monorepo/issues/10432

### CODE REVIEW

File: packages/snap/src/index.ts [hunk 1]
```diff
@@ -240,8 +247,8 @@ async function getOptimalFeeCurrency(tx: CeloTransactionRequest, wallet: CeloWal
   const gasLimit = (await wallet.estimateGas(tx)).mul(5)
   const celoBalance = await wallet.getBalance();
   const tokenAddresses = await feeCurrencyWhitelistContract.getWhitelist();
-
-  if (gasLimit.add(tx.value ?? 0) >= celoBalance) {
+  const GasLimitPlusValue = gasLimit.add(tx.value ?? 0);
```
Comment (aaronmgdr) [inline packages/snap/src/index.ts:250]: nit but its weird to have. variable be Capitalized. ususally this is only for classes in js

File: packages/snap/src/index.ts [hunk 2]
```diff
@@ -139,6 +145,8 @@ export const onRpcRequest: OnRpcRequestHandler = async ({ origin, request }) =>
             },
           });
         }
+      } else { 
+        return;
```
Comment (aaronmgdr) [inline packages/snap/src/index.ts:149]: why did you add this?

File: packages/snap/src/index.ts [hunk 3]
```diff
@@ -106,6 +110,8 @@ export const onRpcRequest: OnRpcRequestHandler = async ({ origin, request }) =>
             overrideFeeCurrency,
             network.name,
           );
+        } else {
+          return;
```
Comment (aaronmgdr) [inline packages/snap/src/index.ts:114]: so if  no  overrideFeeCurrency  is given it will return here and not send the transaction

File: packages/snap/src/index.ts (hunk 2 above)
Comment (aaronmgdr) [reply packages/snap/src/index.ts:149]: oh maybe add a little comment

File: packages/snap/src/index.ts (hunk 3 above)
Comment (montera82) [reply packages/snap/src/index.ts:113]: Yes updated now.

File: packages/snap/src/index.ts (hunk 2 above)
Comment (montera82) [reply packages/snap/src/index.ts]: i can add a comment, but if we follow the if clause we see that we see that we return if user confirmation is false ( i.e if result === false )

File: packages/snap/src/index.ts (hunk 1 above)
Comment (montera82) [reply packages/snap/src/index.ts]: yes agreed...just coding at 11pm 😴

File: packages/snap/src/index.ts (hunk 3 above)
Comment (montera82) [reply packages/snap/src/index.ts:113]: basically 
Disapprove = null
Approve w/o inputing any value = undefined
Approve with inputing a value = value

image

**Review by aaronmgdr — APPROVED** [review]

**Review by pahor167 — APPROVED** [review]
